Triphouse-project/index.html

```diff
@@ -1,33 +1,35 @@
 <!DOCTYPE html>
 <html lang="en">
```

Comment:
![2021-04-22_21h48_26](https://user-images.githubusercontent.com/19935417/115770375-495a6100-a3b5-11eb-96f2-badaef5fa06d.png)

White line on the top ?
Text should be aligned to left.
Padding before logo ?